system/jlib/jptree.cpp

```diff
@@ -8895,7 +8895,10 @@ static std::tuple<std::string, IPropertyTree *, IPropertyTree *> doLoadConfigura
     else
     {
         if (legacyFilename && checkFileExists(legacyFilename))
+        {
             delta.setown(createPTreeFromXMLFile(legacyFilename, ipt_caseInsensitive));
+            newGlobalConfig.set(delta->queryPropTree("global"));
```

Comment:
i.e. always check for a 'global' section in bare-metal configuration if loading one.